Eatery/Onboarding/Controllers/OnboardingViewController.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  OnboardingViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/11/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol OnboardingViewControllerDelegate {
+    func onboardingViewControllerDidTapNext(_ viewController: OnboardingViewController)
+}
+
+class OnboardingViewController: UIViewController {
+
+    internal var stackView: UIStackView!
```

Comment:
Get rid of `internal`: things are internal by default so this just adds clutter.